ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/AmountMapper.java

```diff
@@ -45,6 +45,10 @@ public CCDAmount to(Amount amount) {
 
     @Override
     public Amount from(CCDAmount ccdAmount) {
+        // After data migration remove this null check and enforce presence
```

Comment:
What do we need that for? Amount should always exists.

Reply:
this may be one we can get away without, but in my local I have cases without it.
Probably going to defer merging this until we have some sort of agreement on how we do the data migrations

Reply:
Would be great to investigate why you have it locally and whether we have the same in prod. The only legitimate case I can see right now is the scenario where we changed the field names without reflecting such change in existing data in CCD that causes 'Unrecognised field' error.